### PR TITLE
Add with_counts to fetch_guild

### PR DESCRIPTION
**Pull request by apple502j**
### Summary
This adds discord/discord-api-docs#1528 support.

### Checklist

- [ ] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/guild.py [hunk 1]
```diff
@@ -290,6 +303,8 @@ def _from_data(self, guild):
         self.preferred_locale = guild.get('preferred_locale')
         self.discovery_splash = guild.get('discovery_splash')
         self._rules_channel_id = utils._get_as_snowflake(guild, 'rules_channel_id')
+        self.approximate_presence_count = data.get('approximate_presence_count')
```
Comment (mdashlw) [inline discord/guild.py:306]: same with member count
```suggestion
        self.approximate_presence_count = guild.get('approximate_presence_count')
```

**Comment by JDJGInc** [comment]
It's here:
image
It's also probably better than having to make a manually invite in order to get these.
Source: https://discord.com/developers/docs/resources/guild
Can you also add other missing kwargs from https://discord.com/developers/docs/resources/guild?
if so thanks :D

**Comment by JDJGInc** [comment]
Then again, I think these two values are fine.

**Comment by Rapptz** [comment]
This PR's too out of date, I think if someone wanted to work on this again they can open a new PR.

**Pull request closed without merging**